web/package.json

```diff
@@ -1,58 +1,53 @@
 {
   "name": "client",
   "version": "0.1.0",
+  "engine": ">=22",
   "private": true,
-  "proxy": "http://127.0.0.1:5000",
   "dependencies": {
     "@codemirror/lang-json": "^6.0.1",
-    "@emotion/react": "^11.7.0",
-    "@emotion/styled": "^11.6.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
-    "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/react-fontawesome": "^0.1.16",
-    "@mui/icons-material": "^5.2.0",
-    "@mui/material": "^5.2.2",
-    "@mui/styles": "^5.2.2",
-    "@types/capitalize": "^2.0.0",
-    "@types/lodash": "^4.14.177",
-    "@types/mui-datatables": "^3.7.7",
-    "@types/node": "^14.17.34",
-    "@types/react": "^17.0.37",
-    "@types/react-dom": "^17.0.11",
-    "@types/react-responsive": "^8.0.4",
-    "@types/react-router-dom": "^5.3.2",
-    "@types/use-persisted-state": "^0.3.0",
-    "@uiw/react-codemirror": "^4.19.9",
-    "ansi-regex": "^6.0.1",
-    "autosuggest-highlight": "^3.2.0",
+    "@emotion/react": "^11.14.0",
+    "@emotion/styled": "^11.14.0",
```

Comment:
these are peer dependencies for mui now. peer dependencies are confusing because `yarn` yells above lots of them that are actually fine (or not even fixable because the actual fix is a relaxing of the constraint upstream) but this one is actually required and you get a weird import failure if it is missing.